go/ct/spc/specification_test.go

```diff
@@ -101,10 +101,14 @@ func TestSpecification_EachRuleProducesAMatchingTestCase(t *testing.T) {
 	}
 }
 
-func TestSpecification_SstoreWithTooLittleGasProducesMatchingTestCases(t *testing.T) {
+// the following test has been used to verify stability of a specific rule.
+// so far it has been used for:
+// - sstore_with_too_little_gas_
+// - pc_on_data_is_ignored
+func TestSpecification_SpecifiedRuleProducesMatchingTestCases(t *testing.T) {
 	rnd := rand.New(0)
 	rules := Spec.GetRules()
-	filter := regexp.MustCompile("sstore_with_too_little_gas_")
+	filter := regexp.MustCompile("sstore_with_too_little_gas_Berlin_StorageDeleted_cold")
```

Comment:
How about making this a table-driven test where you list the rules this test should cover?

This way, you would preserve the coverage of the old rule, add coverage for the new rule, and make future re-use easy.